packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -1,29 +1,50 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Should the contract version be incremented?

Reply:
Once this PR will be merged to this branch, it will update the version correctly. https://github.com/celo-org/celo-monorepo/pull/10904